mergin/test/test_client.py

```diff
@@ -303,3 +303,29 @@ def toggle_geodiff(enabled):
         assert not mp3.geodiff.has_changes(mp.fpath_meta('diff'))
     else:
         assert os.path.exists(mp.fpath('base.gpkg_conflict_copy'))
+
+
+def test_list_of_push_changes(mc):
+    PUSH_CHANGES_SUMMARY = "{'base.gpkg': {'geodiff_summary': [{'table': 'gpkg_contents', 'insert': 0, 'update': 1, 'delete': 0}, {'table': 'simple', 'insert': 1, 'update': 0, 'delete': 0}]}}"
+    def toggle_geodiff(enabled):
+        os.environ['GEODIFF_ENABLED'] = str(enabled)
+
+    test_project = 'test_list_of_push_changes'
+    project = API_USER + '/' + test_project
+    project_dir = os.path.join(TMP_DIR, test_project)  # primary project dir for updates
+
+    cleanup(mc, project, [project_dir])
+    # create remote project
+    toggle_geodiff(True)
+    shutil.copytree(TEST_DATA_DIR, project_dir)
+    mc.create_project(test_project, project_dir)
+
+    f_updated = 'base.gpkg'
+    mp = MerginProject(project_dir)
+
+    shutil.copy(mp.fpath('inserted_1_A.gpkg'), mp.fpath(f_updated))
+
+    pull_changes, push_changes, push_changes_summary = mc.project_status(project_dir)
```

Comment:
it's just a matter of styling, but usually if you are not interested in some variables, you can do `_,  _, push_changes_summary = ...`